tests/components/test_conversation.py

```diff
@@ -171,24 +167,49 @@ def test_turn_on_intent(hass, sentence):
     assert call.data == {'entity_id': 'light.kitchen'}
 
 
-@asyncio.coroutine
+async def test_cover_intents_loading(hass):
```

Comment:
@balloob This makes sense?

Reply:
Yes. You will need to setup the conversation component though.

Reply:
Hmm, odd that the test works then since it raises the exception then doesn't after setting up cover.